app/dashboard/static/js/app/view-tests-suite-id.2017.7.2.js

```diff
@@ -0,0 +1,557 @@
+/*!
+ * kernelci dashboard.
+ * 
+ * Copyright (C) 2014, 2015, 2016, 2017  Linaro Ltd.
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+require([
+    'jquery',
+    'utils/init',
+    'utils/error',
+    'utils/request',
+    'utils/urls',
+    'utils/bisect',
+    'tables/test',
+    'utils/html',
+    'utils/const',
+    'utils/format',
+    'utils/date'
```

Comment:
This is an old habit, but you can move the `utils/date` import (or any imports that does not return anything) into the `js/kci-*` file that `require`s this one.